cli/query/query.go

```diff
@@ -0,0 +1,56 @@
+package query
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/go-resty/resty/v2"
+	"github.com/nextdotid/proof-server/config"
+	"github.com/spf13/cast"
+	"os"
+)
+
+type QueryParams struct {
+	Platform string `json:"platform"`
+	Identity string `json:"identity"` // Identity on target platform.
+	Page     int    `json:"page"`
+}
+
+func QueryProof() {
+	config.InitCliConfig()
+	params := initParams()
+	req := make(map[string]string)
+	req["platform"] = params.Platform
+	req["identity"] = params.Identity
+	req["page"] = cast.ToString(params.Page)
+
+	url := getQueryUrl()
+	client := resty.New()
+	resp, err := client.R().SetQueryParams(req).EnableTrace().Get(url)
+
+	fmt.Println(resp)
```

Comment:
You can't just `stdout` the response body, the JSON struct in body is a mess, and not suitable for human to read.

See  comment (L44) below.